sdk/src/fee.rs

```diff
@@ -82,18 +82,13 @@ impl FeeStructure {
         message: &SanitizedMessage,
         lamports_per_signature: u64,
         budget_limits: &FeeBudgetLimits,
-        remove_congestion_multiplier: bool,
         include_loaded_account_data_size_in_fee: bool,
     ) -> u64 {
         // Fee based on compute units and signatures
         let congestion_multiplier = if lamports_per_signature == 0 {
```

Comment:
which tests are affected by this remaining branch? Seems unecessary to have this branch with the concept of congestion multiplier removed.